docs/src/inverse_problems/petab_ode_param_fitting.md

```diff
@@ -0,0 +1,487 @@
+
+## [Additional features: Simulation options](@id petab_simulation_options)
+
+While in our basic example, we do not provide any additional information to our `PEtabODEProblem`, this is an opportunity to specify how the model should be simulated, and what automatic differentiation techniques to use for the optimisation procedure (if none are provided, appropriate defaults are selected). 
+
+Here is an example, taken from the [more detailed PEtab.jl documentation](https://sebapersson.github.io/PEtab.jl/dev/Boehm/#Creating-a-PEtabODEProblem)
+```petab1
+petab_problem = PEtabODEProblem(petab_model, 
+                                ode_solver=ODESolver(Rodas5P(), abstol=1e-8, reltol=1e-8), 
+                                gradient_method=:ForwardDiff, 
+                                hessian_method=:ForwardDiff)
+nothing # hide
+```
+where we simulate our ODE model using the `Rodas5p` method (with absolute and relative tolerance both equal `1e-8`) and use [forward automatic differentiation](https://github.com/JuliaDiff/ForwardDiff.jl) for both gradient and hessian computation. More details on available ODE solver options can be found in [the PEtab.jl documentation](https://sebapersson.github.io/PEtab.jl/dev/API_choosen/#PEtab.ODESolver).
+
+## [Additional features: Optimisation](@id petab_optimisation)
+
+### [Optimisation methods and options](@id petab_optimisation_optimisers)
+For our examples, we have used the `Optim.IPNewton` optimisation method. PEtab.jl supports [several additional optimisation methods](https://sebapersson.github.io/PEtab.jl/dev/Avaible_optimisers/#options_optimisers). Furthermore, `calibrate_model`'s `options` argument permits the customisation of the options for any used optimiser. E.g. to designate the maximum number of iterations of the `Optim.IPNewton` method we would use:
+```petab1
+res = calibrate_model(petab_problem, p0, IPNewton(); options=Optim.Options(iterations = 10000))
+nothing # hide
+```
+
+Please read the [PEtab.jl documentation](https://sebapersson.github.io/PEtab.jl/dev/Avaible_optimisers/#options_optimisers) to learn how to customise the various optimisers' properties. 
+
+### [Optimisation path recording](@id petab_optimisation_path_recording)
+
+To record all the parameter sets evaluated (and their objective values) during the optimisation procedure, provide the `save_trace=true` argument to `calibrate_model` (or `calibrate_model_multistart`). This is required for the various [optimisation evaluation plots](@ref petab_plotting) provided by PEtab.jl. If desired, this information can be accessed in the calibration output's `.xtrace` and `.ftrace` fields.
+
+
+## Objective function extraction
+While PEtab.jl provides various tools for analysing the objective function generated by `PEtabODEProblem`, it is also possible to extract this function for customised analysis. Given a `PEtabODEProblem`
+```petab1
+petab_problem = PEtabODEProblem(petab_model)
+nothing # hide
+```
+We can find the:
```

Comment:
I think it might be worth to mention here that lower and upper bounds can be accessed from the petab_problem, as well as the order of parameters via `petab_problem.θ_names`

Reply:
This part was really only meant if the user wanted to solely use PEtab to generate cost functions (and then optimize these using SciML/Optimization) or use for something else, rather than listing the fields of the problem. Although this should probably be somewhere (but I might leave that out from this tutorial).

Reply:
Yes, but then I agree it should be left at it is